fsspec/implementations/cached.py

```diff
@@ -128,7 +150,7 @@ def _strip_protocol(path):
             # acts as a method, since each instance has a difference target
             return self.fs._strip_protocol(type(self)._strip_protocol(path))
 
-        self._strip_protocol = _strip_protocol
+        self._strip_protocol = _strip_protocol  # type: ignore[method-assign]
```

Comment:
I don't know how to make mypy happy usually.

```suggestion
        self._strip_protocol: Callable = _strip_protocol  # works ?
```